.moban.yaml

```diff
@@ -1,6 +1,30 @@
 overrides: coala.yaml
 
 name: coala-quickstart
+contact: coala.analyzer@gmail.com
```

Comment:
I thought we had `@coala.io` domain e-mails

Reply:
This PR is not to change the contents of setup.py .  These values are just copies of what is in there.